pkg/steps/source.go

```diff
@@ -242,7 +242,10 @@ func handleBuild(buildClient BuildClient, build *buildapi.Build, dry bool, artif
 				return fmt.Errorf("could not get build %s: %v", build.Name, err)
 			}
 
-			if isInfraReason(b.Status.Reason) || hintsAtInfraReason(b.Status.LogSnippet) {
```

Comment:
Can you make a helper `isTerminated` for readability here

Reply:
only under protest.  one exists in origin but you don't have a dep on origin.

Reply:
updated.

Reply:
I remember making an issue for you guys to expose your helpers in an importable way like a year ago ;)

Reply:
what's not importable about origin, hmmmmmm?